hedera-node/src/main/java/com/hedera/services/state/exports/SigFileWriter.java

```diff
@@ -22,5 +22,9 @@
 
 @FunctionalInterface
 public interface SigFileWriter {
+
+	byte TYPE_SIGNATURE = 3;       // the file content signature, should not be hashed
+	byte TYPE_FILE_HASH = 4;       // next 48 bytes are hash384 of content of the file to be signed
```

Comment:
Wouldn't it be better to make them `final` too?

Reply:
we don't have to use `final` in interfaces